Furball.Vixie.TestApplication/Tests/TestFSS.cs

```diff
@@ -38,18 +38,18 @@ public class TestFSS : GameComponent {
             this._quadRendererGl.Begin();
             this._quadRendererGl.DrawString(this._font, "VixieFontStashSharpRenderer Testing",                new Vector2(10,  100), Color.White,      this._rotation,                         new Vector2(_scale));
             this._quadRendererGl.DrawString(this._font, "More Quite Long Text, Foxes Are Cute",               new Vector2(10,  200), Color.LightBlue,  (float)(2f * Math.PI - this._rotation), new Vector2(_scale));
-            this._quadRendererGl.DrawString(this._font, "Did You Know That A Bee Should Not Be Able To Fly?", new Vector2(500, 300), Color.LightGreen, (float)(Math.PI - this._rotation),       new Vector2(_scale));
+            this._quadRendererGl.DrawString(this._font, "Did You Know That A Bee Should Not Be Able To Fly?", new Vector2(50, 300), Color.LightGreen, (float)(Math.PI - this._rotation),       new Vector2(_scale));
```

Comment:
This position should go back to the old one

Reply:
i forgot what the old one was xd